docs/cave/bundles-and-procedures.md

```diff
@@ -1,48 +1,48 @@
-# Bundles and Procedures
-AWIPS contains two methods for saving and loading data resources: **Bundles** are a simple way to save loaded resources to access in future CAVE sessions.  **Procedures** are similar to Bundles, but can be thought of as *groups of bundles* and allows the user to manage saved resources with more control.
+# Displays and Procedures
+AWIPS contains two methods for saving and loading data resources: **Displays** are a simple way to save loaded resources into a bundle file to access in future CAVE sessions.  **Procedures** are similar to Displays, but can be thought of as *groups of displays* and allows the user to manage saved resources with more control.
 
 
-##  Bundles
+##  Displays
 
 ### **File > Load Display**
 
-Load a previously-saved bundle from within the AWIPS system.  The Open Bundle dialog allows you to select your own saved bundles as well as those saved by other users.
+Load a previously-saved display from within the AWIPS system.  The pop-up dialog allows you to select your own saved displays as well as those saved by other users.
 
-Each selected bundle will load its contents to new tabs which are named after the bundle file name (e.g. NAM_ThetaE).
+Each selected display will load its contents to new tabs which are named after the display file name (e.g. GFS_TempWind).
 
-![image](../images/KTZZHoCV5S.gif)
+![image](../images/load-display.gif)
 
-Most saved bundles will consist of a single Map Editor (tab), but with multiple tabs saved each will open again in its own Map Editor.
+Most saved displays will consist of a single Map Editor (tab), but with multiple tabs saved each will open again in its own Map Editor.
 
 ---
 
-### **File > Save Display**
-
-Save a product display within the AWIPS system, synching the bundle between CAVE and the EDEX server.
+### **Load Display from Local Disk**
 
----
+To load a previously-saved display from a path within the file directory of the workstation, select **File > Load Dislay** and then select the **File** button on the right to browse your local directories.
 
-### **File > Manage Bundles**
+![image](../images/load-local-display.gif)
 
-Select and remove a saved bundle under File > Manage Bundles, this will open the Delete Bundle dialog.  Select the file name and click **OK** and then confirm deletion to remove the saved file permanently.
+---
 
-![image](../images/fileMenuManageBundles.png)
+### **File > Save Display**
 
-![image](../images/delete_bundle2.png)
+Save a product display within the AWIPS system. This syncs the display file between CAVE and the EDEX server.
```

Comment:
This second sentence is a bit confusing... "Syncs the display file between CAVE and the EDEX server".  Do you mean it saves the current display to the edex server for the specific user?
The word "sync" to me implies that the display on cave and edex is the same... which doesn't really make sense because edex doesn't have a display so to speak.

Reply:
This is the "display file" and it is the same in caveData on the local CAVE machine and also on the EDEX server for the specific user in the utility tree.

Reply:
Yes, whatever the name of the xml file, it saves it to caveData and to EDEX. If there are multiple display files then it will save those as well.
```
(base) [awips@localhost ~]$ find caveData -name GFS*
caveData/common/user/awips@localhost.localdomain/perspectives/GFS-TempWind.xml
```

Reply:
Okay I think it would make most sense to users then to just say 

> Save a product display within the AWIPS system.  This saves the display both locally and to the EDEX server for your specific user.

Or something like that, instead of using the word sync.

Reply:
Okay sounds good.